### PR TITLE
ci: add gh-actions config

### PR DESCRIPTION
**Pull request by eltongarbin**
### Resources

- https://github.com/ocean-ds/ocean-android/blob/master/.github/workflows/ci.yml
- https://docs.github.com/en/actions/guides/building-and-testing-swift
- https://augmentedcode.io/2021/04/26/running-tests-in-swift-package-with-github-actions/
- https://about.codecov.io/blog/code-coverage-for-ios-development-using-swift-xcode-and-github-actions/
- https://kazaimazai.com/ci-cd-with-github-actions-ios-swiftlint-fastlane/

### CODE REVIEW

**Review by vromeiro — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          fetch-depth: 0
+
+      - name: Select Xcode Version
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest-stable
+    
+      - name: Bundle install
+        run: |
+          bundle install
```
Comment (eltongarbin) [inline .github/workflows/ci.yml:22]:
```suggestion
        run: bundle install
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        
+      - name: Install Pods
+        run: |
+          pod install
```
Comment (eltongarbin) [inline .github/workflows/ci.yml:26]:
```suggestion
        run: pod install
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            
+      - name: Run tests 
+        run: |
+          bundle exec fastlane build_test
```
Comment (eltongarbin) [inline .github/workflows/ci.yml:30]:
```suggestion
        run: bundle exec fastlane build_test
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        uses: actions/checkout@v2
+        with:
+          # we actually need "github.event.pull_request.commits + 1" commit
+          fetch-depth: 0
+
+      - name: Select Xcode Version
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest-stable
+
```
Comment (eltongarbin) [inline .github/workflows/ci.yml:19]: All projects of this org enforce a conventional commits. So, include this job to guaranteed that.
```
      - name: Lint commit message
        uses: wagoid/commitlint-github-action@v3
```

**Review by vromeiro — APPROVED** [review]

**Pull request merged**